onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -1061,5 +1061,7 @@
     <!-- Bikeshare preference option -->
     <string name="bike_info_window_spaces_title">Spaces</string>
     <string name="layers_speedial_bikeshare_label">Bikeshare</string>
+    <string name="bike_station_title">Rent a bike</string>
```

Comment:
- [ ] Let's change `bike_station_title` to `Rent a bike here` to better indicate it's related to this station.